.golangci.yml

```diff
@@ -45,7 +45,7 @@ linters-settings:
   lll:
     line-length: 120
   gocyclo:
-    min-complexity: 24
+    min-complexity: 26
```

Comment:
Instead of changing `min-complexity` please simplify the code that you've added.

Reply:
Do you have any suggestion on how to simplify the code? As I only added a for loop and then the complexity goes up.

Reply:
Usually this is done by adding helper functions and using them.